src/Microsoft.AspNet.OData.Shared/Query/ODataQueryOptions.cs

```diff
@@ -382,6 +387,16 @@ public virtual IQueryable ApplyTo(IQueryable query, ODataQuerySettings querySett
                 result = orderBy.ApplyTo(result, querySettings);
             }
 
+            if (IsAvailableODataQueryOption(SkipToken, AllowedQueryOptions.SkipToken))
```

Comment:
>SkipToken, AllowedQueryOptions.SkipToke [](start = 44, length = 39)

do we need the 'StableSortOrderBy' for the skiptoken?

Reply:
Yes. We need the OrderByNodes/Clause as they were applied to the data. 

---
In reply to: [282610676](https://github.com/OData/WebApi/pull/1724#discussion_r282610676) [](ancestors = 282610676)